### DISCUSSION TITLE
能不能搞一个合并发送捏

### DISCUSSION THREAD

**Discussion by cdmc00** [koishijs/koishi-plugin-booru#100] [Ideas]
能不能搞个合并发送，这样在QQ里反而更方便捏

**Comment by MaikoTan** [comment]
The forward messages feature is already implemented in v1.0.3 and later when the bot is sending more than one image. Is there any problem you encountered with it?

当机器人发送多张图像时，转发消息功能已在 v1.0.3 及更高版本中实现。您在使用过程中遇到什么问题吗？

**Reply by cdmc00** [reply]
单张+详情的话不可以嘛？

**Reply by MaikoTan** [reply]
I think it would make every user one more clicking before viewing the image, decrease the user experience. If you insist this change, try to persuade me.

我认为这会让每个用户在查看图像之前点击更多一次，降低用户体验。如果你坚持这个改变，请尝试说服我。

**Reply by cdmc00** [reply]
避免一些群友点进群看到NSFW而社死，并且套壳有利于不被群主撤回（毕竟直接涩图撤回概率为1000%）

**Reply by MaikoTan** [reply]
I think I could add the `spoiler` feature then, which would be implemented as forward messages on QQ.

我想我可以添加“剧透”功能，该功能在QQ将作为转发消息实现。

**Reply by cdmc00** [reply]
cool  期待

**Comment by MaikoTan** [comment]
This feature request is converted into an issue [here](https://github.com/koishijs/koishi-plugin-booru/issues/114) and @Lipraty is working on it. I think this discussion can be closed.
